### PR TITLE
I am pretty sure it's 2014 

### PR DESCRIPTION
**Pull request by ghost**
Copyright year changed to 2014 :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 [![Coverage
 Status](https://coveralls.io/repos/pythonhub/quokka/badge.png)](https://coveralls.io/r/pythonhub/quokka)
 [![Twitter](http://badgr.co/twitter/@quokkaproject.png?bg=%2302779E)](http://twitter.com/quokkaproject)
+[![Gitter chat](https://badges.gitter.im/itsnauman/quokka.png)](https://gitter.im/itsnauman/quokka)
```
Comment (rochacbruno) [inline README.md]: Nice idea! But I think this line should be https://gitter.im/pythonhub/quokka

Comment (ghost) [inline README.md]: Wait let me check it :smiley:

Comment (ghost) [inline README.md]: @rochacbruno It's not possible, you need to make a chat room while authenticated from Python Hub :smile:

Comment (rochacbruno) [inline README.md]: Already created take a look https://gitter.im/pythonhub/quokka

Comment (ghost) [inline README.md]: Thats cool

**Comment by ghost** [comment]
I removed the gitter.im banner from my fork

**Comment by ghost** [comment]
Thanks :thumbsup:

**Pull request merged**